tests/invalid/DefiniteAssign_Invalid_4.whiley:5: unable to resolve name (x)
    f(x)
      ^
